spec/duration.html

```diff
@@ -1489,7 +1489,7 @@ <h1>
         1. If _largestUnit_ is not one of *"year"*, *"month"*, *"week"*, or *"day"*, then
           1. Let _diffNs_ be ! DifferenceInstant(_relativeTo_.[[Nanoseconds]], _endNs_, 1, *"nanosecond"*, *"halfExpand"*).
```

Comment:
Maybe it would be helpful to document your reasoning in-spec, like with "Assert: abs(_diffNs_) ≤ 1.728 × 10<sup>22</sup>."

Reply:
Good idea.